src/ol/format/gmlbase.js

```diff
@@ -31,6 +31,7 @@ goog.require('ol.xml');
  * ol.format.GML3.
  *
  * @constructor
+ * @abstract
```

Comment:
Also surprised to see `@abstract` on a function that has code that looks like it should be executed.